components/tools/OmeroWeb/omeroweb/webclient/show.py

```diff
@@ -860,12 +860,12 @@ def paths_to_well_image(conn, params, well_id=None, image_id=None,
                acquisition.id,
                well.id
         from WellSample wellsample
-        left outer join wellsample.details.owner wsowner
+        join wellsample.details.owner wsowner
         left outer join wellsample.plateAcquisition acquisition
-        left outer join wellsample.details.owner aowner
+        left outer join acquisition.details.owner aowner
```

Comment:
should be a ``join``?
Do you need the owner for plate/plate acquisition/well sample?
It is unlikely to be different.